src/routes/api/conversations/+server.ts

```diff
@@ -0,0 +1,15 @@
+import { collections } from "$lib/server/database";
+
+export async function GET({ locals }) {
+	if (locals.user?._id || locals.sessionId) {
+		const res = await collections.conversations
+			.find({
+				...(locals.user ? { userId: locals.user._id } : { sessionId: locals.sessionId }),
```

Comment:
I think if you want to find only the relevant user conversations you can use `authCondition(locals)` and import `authCondition` from `"$lib/server/auth"`, it's the one we use everywhere to keep it centralized 😁

Reply:
on it.